LION/data_loaders/deteCT.py

```diff
@@ -423,6 +423,9 @@ def __getitem__(self, index):
             # Get paths to the dataset
             target = self.__load_and_preprocess_segmentation__(index)
 
+        if self.task == "groundtruth":
+            return target
```

Comment:
Should we `return None, target` ? What do you think?

Reply:
I think PyTorch will complain if we do that, because then the dataloader tries to collate the `None`s (so we would need a custom `collate_fn` when using the PyTorch data loaders):
![Screenshot 2024-05-02 at 11 14 46](https://github.com/CambridgeCIA/LION/assets/12610714/6410ad0a-5962-41b8-9fcc-7dff6743f973)
I also like being able to just do `for x in loader`. What would be the reason for `return None, target`?

Reply:
@fsherry ah makes sense. Was mostly thinking on consistent outputs for the dataloader, but its not worth the hassle.